.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    name: MacOS
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Homebrew Update
+      run: brew update
```

Comment:
This adds a minute to builds (and was something we avoided on Circle), but without it we got the same failure as Circle 🤷‍♂

Reply:
huh dang. wonder if we could speed it up by saving some cache artifacts or something. something to look into later